docs/source/.gitignore

```diff
@@ -0,0 +1 @@
+gallery/
```

Comment:
Is this remnant?

Reply:
No, this is to avoid generated `gallery/` folder to be included in our git repo, as all the source files should be stay in `_gallery_original`